Makefile.am

```diff
@@ -53,15 +59,21 @@ stream_DATA = $(NULL)
 recordingsdir = $(datadir)/janus/recordings
 recordings_DATA = $(NULL)
 
+if WINDOWS_OS
+INSTDIR = ../..
```

Comment:
Well, it does entirely make sense if you want to use 3rd party installer on Windows which is more common approach in that OS. I agree that paths could be changed after compilation but this make very limited use case on Windows. I also enabled relative path to the main config. Please refer to (https://github.com/marcin-sielski/janus-gateway/releases) binaries. Windows environment is by nature is slightly diffrent then Linux. Hope that helps.